src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -28,14 +30,39 @@
      * 26.04.2019 Сергей 7 100
      * 26.04.2019 София 9 100
      * 26.04.2019 Сергей 11 50
-     *
+     * <p>
      * Пример вывода:
      * Отчёт за период 01.04.2019  - 30.04.2019
      * Сергей - 1550
      * Андрей - 600
      * София - 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        if (Integer.parseInt(dateTo.substring(0, 2))
```

Comment:
@NikolayKatsai what if you will compare dateFrom = 14.07.2019 and dateTo = 10.08.2019?